other-docs/guides/updating-php/README.md

```diff
@@ -9,9 +9,14 @@ There are 2 key steps to getting ready for a new version of PHP:
 
 ## Altis Compatibility Chart
 
+**Note:** PHP 8.0 will officially be available from the Altis v12 release date onwards. Altis v11 will run on PHP 8.0 but support is experimental. Keep reading to learn how to prepare your application for a new PHP version.
+
 |Altis|PHP Version|
```

Comment:
@roborourke @missjwo Might be worth expanding this out a bit for clarity?

| Altis  | PHP 7.4        | PHP 8.0        |
| ------ | -------------- | -------------- |
| v13    |                | **Supported**  |
| v12    | *Deprecated*   | **Supported**  |
| v11    | **Supported**  | *Experimental* |
| v10    | **Supported**  |                |
| v9     | **Supported**  |                |
| v8     | **Supported**  |                |
| v7     | **Supported**  |                |